code/modules/events/prison_break.dm

```diff
@@ -3,38 +3,52 @@
 	announceWhen	= 75
 
 	var/releaseWhen = 60
-	var/list/area/areas = list()		//List of areas to affect. Filled by start()
+	var/list/area/areas = list()		// List of areas to affect. Filled by start()
 
-	var/eventDept = "Security"			//Department name in announcement
-	var/list/areaName = list("Brig")	//Names of areas mentioned in AI and Engineering announcements
-	var/list/areaType = list(/area/security/prison, /area/security/brig)	//Area types to include.
-	var/list/areaNotType = list()		//Area types to specifically exclude.
+	// Chosen from a random prisonbreakarea datum on setup()
+	var/eventDept					// Department name in announcement
+	var/areaName					// Name of areas mentioned in AI and Engineering announcements
+	var/list/areaType = list()		// Area types to include.
+	var/list/areaNotType = list()	// Area types to specifically exclude.
 
 /datum/event/prison_break/setup()
-	announceWhen = rand(75, 105)
-	releaseWhen = rand(60, 90)
+
+	var/d = pick(subtypesof(/datum/prisonbreakarea))
+	var/datum/prisonbreakarea/D = new d
```

Comment:
weird var names

Reply:
What var names do you suggest?

Reply:
something more understandable, instead of small d and big D.